docs/guides/external-node/08_pruning.md

```diff
@@ -0,0 +1,38 @@
+# Pruning
+
+It is possible to configure Node to periodically remove all data from batches older than a threshold, both from Postgres
+and from tree.
+
+> [!NOTE]
+>
+> If you need a node with data retention period of up to a few days, please set up a node from a snapshot (see previous
+> chapter) and wait for it to have enough data. Pruning an archival node can take unpractical amount of time. In the
+> future we will be offering pre-pruned DB snapshots with a few months of data.
+
+You can enable pruning by setting
```

Comment:
Similar to *Snapshot Recovery*, it's worth clarifying "...by setting the environment variable".

Reply:
corrected